image/distributionutil/registry.go

```diff
@@ -29,6 +29,7 @@ func NewV2Repository(named reference.Named, actions ...string) (distribution.Rep
 	authConfig, err := utils.GetDockerAuthInfoFromDocker(named.Domain())
 	if err != nil {
 		logger.Warn("failed to get auth info, err: %s", err)
+		return nil, err
```

Comment:
Sometimes we don't need login the registry when we build CloudImage if Kubefile only has COPY command. @kakaZhou719 double check this PR